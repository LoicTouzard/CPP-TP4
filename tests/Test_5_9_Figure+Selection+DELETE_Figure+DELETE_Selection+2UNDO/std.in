C cercle 0 0 1
S select -1 1 1 -1
DELETE cercle
DELETE select
UNDO
UNDO
LIST
MOVE select 2 2
EXIT
