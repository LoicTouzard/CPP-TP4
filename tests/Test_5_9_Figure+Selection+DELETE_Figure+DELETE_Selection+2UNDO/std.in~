C c 0 0 1
S s -1 1 1 -1
MOVE s 1 1
UNDO
UNDO
REDO
REDO
LIST
MOVE s 1 1
LIST
EXIT

