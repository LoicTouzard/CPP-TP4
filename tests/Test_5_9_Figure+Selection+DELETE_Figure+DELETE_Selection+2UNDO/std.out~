OK
#New object :c
OK
#New object : s
OK
OK
OK
OK
OK
C c 1 1 1
OK
C c 2 2 1

