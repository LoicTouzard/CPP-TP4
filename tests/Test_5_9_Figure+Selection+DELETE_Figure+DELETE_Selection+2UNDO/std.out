OK
#New object :cercle
OK
#New object : select
OK
OK
OK
OK
C cercle 0 0 1
ERR
#select not found

